docs/source/code_examples/tutorials/end-to-end/preparing_data_for_labelling.py

```diff
@@ -21,8 +21,10 @@
 #%%
 # Authenticating
 # --------------
-# To interact with Encord, you need to authenticate a client. You can find more details
-# :ref:`here <authentication:Authentication>`.
+# .. note::
+#
+#   To interact with Encord, you need to authenticate a client. You can find more details
+#   :ref:`here <authentication:Authentication>`.
```

Comment:
Maybe a link to the specific section: http://localhost:63344/cord-client-python/docs/_build/html/authentication.html#user-authentication